mutest.gemspec

```diff
@@ -39,8 +39,4 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency('anima',         '~> 0.3.0')
   gem.add_runtime_dependency('concord',       '~> 0.1.5')
   gem.add_runtime_dependency('regexp_parser', '~> 0.4.1')
-
-  gem.add_development_dependency('rubocop-rspec', '~> 1.10.0')
```

Comment:
> WDYM?

Eh don't worry about it, I just don't like writing `gem.add_development_dependency` over `gem` when I don't see a reason but thats just personal. Ignore this point

> Well, we already have a development group so it seems weird to be inconsistent and have some gems in one group and others in another.

Yeah I'll be consistent and just remove the group. It isn't even being used for anything. I use pry all the time when debugging tests and we never do like `bundle install --only test`.